iocore/aio/AIO.cc

```diff
@@ -24,7 +24,6 @@
 /*
  * Async Disk IO operations.
  */
-
```

Comment:
can you remove this line removal? It doesn't look related to the slice plugin

Reply:
agreed.  This PR should fix a dumb bug.